Sources/ProjectSpec/Target.swift

```diff
@@ -130,11 +130,13 @@ public struct TargetScheme {
     public var testTargets: [String]
     public var configVariants: [String]
     public var gatherCoverageData: Bool
+    public var commandLineArguments: [String: Bool]
```

Comment:
Would anyone want to specify disabled arguments? In other words is there a need to make this a dictionary as opposed to an array?

Reply:
commandLineArguments or just arguments? What does the Xcode interface use? (on my phone at the moment)

Reply:
- Disabled arguments are actually useful if you have different modes your program can run in but their behavior is conflicting. The advantage is that you could define all the configurations in the schema and let developers toggle them on/off during development
- The element in `xcscheme` is `CommandLineArguments` so it's based off that. The Xcode UI has a more verbose `Arguments passed on launch`

Reply:
Perhaps `launchArguments`? What do you think?

Reply:
I can submit an update for launch arguments

Reply:
Ok cool